docs/product-manuals/clients/java-client/index.md

```diff
@@ -55,14 +55,16 @@ In Java code, instantiate the client as follows:
 
 Let's go over this code snippet line by line:
 
-- First we declare a couple of variables to define the connection properties. These values can be taken from the connection info on the _Client Credentials_ page. Note that `clientSecret` is only visible when you create the client credentials.
-- Next we create the credentials provider for the OAuth protocol. This is needed to authenticate your client.
-- Next we create the client passing in the address of the cluster we want to connect to as well as the credentials provider from the step above.
-- Finally, we send a test request to verify that the connection was established.
+1. First, we declare a few variables to define the connection properties. These values can be taken from the connection information on the **Client Credentials** page. Note that `clientSecret` is only visible when you create the client credentials.
+2. Next, we create the credentials provider for the OAuth protocol. This is needed to authenticate your client.
+3. Next, we create the client by passing in the address of the cluster we want to connect to and the credentials provider from the step above.
+4. Finally, we send a test request to verify the connection was established.
 
-See `io.camunda.zeebe.client.ZeebeClientBuilder` for a description of all available configuration properties.
+See **io.camunda.zeebe.client.ZeebeClientBuilder** for a description of all available configuration properties.
 
-Another, more compact, option is to pass in the connection settings via environment variables:
+[//]:# (Can we link to the location above?)
```

Comment:
https://javadoc.io/doc/io.zeebe/zeebe-client-java/latest/io/zeebe/client/ZeebeClientBuilder.html

Check with @menski to make sure this is what he means.

In general, I don't think we have a good journey into our javadocs from our documentation, so this may be a good opportunity to evaluate how (or if) we make this more obvious.

Reply:
Followed up with @menski for some clarification and he confirmed the link here is appropriate :)